client/API.md

```diff
@@ -0,0 +1,536 @@
+# Kapacitor API Reference Documentation
```

Comment:
This api should be versioned:

`/v1/kapa/tasks` etc

This will ease the transition from your existing api to something else.

Additionally, when one tries to sew many APIs routes into a single domain it is nice if they are namespaced.